app/views/session.py

```diff
@@ -35,15 +36,15 @@ def login():
     if session:
         session.clear()
 
-    logger.debug("attempting token authentication")
-    authenticator.jwt_login(request)
-    logger.debug("token authenticated - linking to session")
+    token = decrypt_token(request.args.get('token'))
+    if not use_token(token.get('jti')):
```

Comment:
In my experience not specifying these things explicitly normally results in 'best case' poor implementation; 'worst case' broken/wrong implementations.